src/fieldViews/AttributeFieldView.ts

```diff
@@ -22,11 +22,8 @@ export abstract class AttributeFieldView<Fields extends unknown>
     // Returns the current position of the parent FieldView in the document.
     private getPos: () => number,
     // The offset of this node relative to its parent FieldView.
-    private offset: number,
-    defaultFields: Fields
-  ) {
-    this.createInnerView(node.attrs.fields || defaultFields);
```

Comment:
Is it worth adding a note on the parent class that inheritors must make this call themselves, now we've removed it from the parent constructor? (Genuine question, maybe we feel it doesn't matter!)

Reply:
Ah yes, definitely.